controllers/datafoundation.go

```diff
@@ -111,6 +118,13 @@ func dfIsReadyToBeRemoved(r *ManagedFusionOfferingReconciler, offering *v1alpha1
 }
 
 func dfReconcile(offeringReconciler *ManagedFusionOfferingReconciler, offering *v1alpha1.ManagedFusionOffering) (ctrl.Result, error) {
+	// Check if the required CRDs are installed or not
+	if !doesCRDExistsAtStart {
+		// we only check for storagecluster CRD because other crds are installed as a dependency via ocs-operator
+		if utils.DoesCRDsExists("storageclusters.ocs.openshift.io") {
+			return ctrl.Result{}, errRestart
```

Comment:
I don't think this is an error. We are restarting because the offering installation succeeded.